bindata/etcd/cluster-backup.sh

```diff
@@ -45,6 +45,10 @@ function backup_latest_kube_static_resources {
     LATEST_RESOURCE_DIRS+=("${LATEST_RESOURCE#${CONFIG_FILE_DIR}/}")
   done
 
+ # Add restore-pod directory the archive
```

Comment:
could we explore adding  {Name: "restore-etcd-pod"}, to`RevisionConfigMaps`[1]

I think that way we just back it up by default. I think then we could consider dropping the ENV var `RESTORE_ETCD_POD_YAML` all together unless we have other reasons to keep it.

[1] https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/starter.go#L271

Reply:
Thanks. Done.